webpack.embedding-sdk-cli.config.js

```diff
@@ -3,18 +3,20 @@ const webpack = require("webpack");
 const TerserPlugin = require("terser-webpack-plugin");
 
 const SDK_CLI_DIST_PATH = path.join(__dirname, "/resources/embedding-sdk/dist");
-
```

Comment:
I kept getting build errors since we didn't have the `embedding-cli` alias so I added that